packages/tailwind-style/dist/tailwind.css

```diff
@@ -846,7 +846,7 @@ Ensure the default browser behavior of the `hidden` attribute.
   color: var(--tw-prose-links);
 }
 
-.prose :where(a:hover):not(:where([class~='not-prose'] *)) {
+.prose :where(.ProseMirror):not(:where([class~='not-prose'] *)) a:hover {
```

Comment:
This tailwind package should be framework agnostic so users not using prosemirror to render the text should not be impacted by any changes made to this package.
I think a better approach is to use the class `className="not-prose"` to remove the styling on the affected blocks.

Reply:
for some reason, adding `className="not-prose"` doesn't seem to remove styling on affected blocks. Looking at the devtools, the text decoration is still appearing so wondering if there is another way to remove this?

Reply:
It definitely should :( could you please push the not working part so I can have a look?